services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/HankeError.kt

```diff
@@ -49,7 +49,7 @@ enum class HankeError(val errorMessage: String) {
     }
 }
 
-class HankeNotFoundException(val hankeTunnus: String) :
+class HankeNotFoundException(val hankeTunnus: String?) :
```

Comment:
When null use alternate error string?

Reply:
If hankeTunnus is null, it logs hanke null not found. So I think it has enough information on what went wrong and why.